crates/core/src/network/partition_processor_rpc_client.rs

```diff
@@ -90,14 +99,17 @@ impl<T> From<RpcError<T>> for PartitionProcessorRpcClientError {
 #[derive(Debug, Clone)]
 pub enum AttachInvocationResponse {
     NotFound,
+    /// Returned when the invocation hasn't an idempotency key, nor it's a workflow run.
```

Comment:
So I can't attach to a running non-idempotent invocation?

Reply:
No, that's the current semantics.

Reply:
Is this contract documented somewhere? In our docs I could only find https://docs.restate.dev/develop/java/clients/#retrieve-result-of-invocations-and-workflows which does not do any distinction between idempotent and non-idempotent invocations. Why wouldn't it be possible to attach to a non-idempotent invocation that I initially submitted via send? Isn't it the same as if I submit an idempotent invocation with retention time of 0?